projects/aca-content/src/lib/dialogs/folder-details/folder-information.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="aca-folder-info-container">
+  <div class="aca-folder-info-header">
+    <div class="aca-folder-icon">
+      <img alt="{{ 'APP.FOLDER_INFO.ICON' | translate }}" src="{{ folderDetails.icon }}">
```

Comment:
Hmm Intellij warns me to use ngSrc directive (introduced by A15) instead of src:
<img width="1083" alt="image" src="https://github.com/user-attachments/assets/edebcb37-5aec-439a-8ba2-02ac724f7570">

@swapnil-verma-gl, @MichalKinas what is your opinion on that, should we use now ngSrc instead of src attribute?

Reply:
Yes this is a very good point and good practice, I would us `ngSrc`

Reply:
Thank you @MichalKinas for opinion. @swapnil-verma-gl please fix this place then. Also @swapnil-verma-gl could you please create jira task to fix all old src places for each project ADF/ACA/ADW/ACC? Please put link to task here when you create it.

Reply:
Replaced src with ngSrc. Also, as part of the replacement, I also had to add a width and height of 24px

Reply:
Created task [ACS-9087](https://hyland.atlassian.net/browse/ACS-9087) for replacing src with ngSrc